app/views/kaminari/_paginator.html.erb

```diff
@@ -0,0 +1,25 @@
+<%# The container tag
+  - available local variables
+    current_page:  a page object for the currently displayed page
+    total_pages:   total number of pages
+    per_page:      number of items to fetch per page
+    remote:        data-remote
+    paginator:     the paginator that renders the pagination tags inside
+-%>
+<%= paginator.render do -%>
+  <nav class="flex items-center justify-between m-10" role="navigation" aria-label="pager">
```

Comment:
if we don't have many pages, it looks a bit odd
![image](https://github.com/Contributees/First-Ruby-Quest/assets/45850166/cbe99bfe-907f-43ec-80aa-637573f8b3ad)

Reply:
what about having the pagination as flex justify-center?

![image](https://github.com/Contributees/First-Ruby-Quest/assets/45850166/7a21d12d-4308-4933-9554-abe3c936957d)

Reply:
Thank you @ClaireDMT! I changed justify-between to justify-center :+1:

With many pages:
![Capture d'écran 2024-07-01 150845](https://github.com/Contributees/First-Ruby-Quest/assets/99920845/705c69c9-1231-457f-9fce-232daf97be39)

![Capture d'écran 2024-07-01 150927](https://github.com/Contributees/First-Ruby-Quest/assets/99920845/7f632d65-5fa9-4a19-b3f3-3a34f05bec1c)

With only a few pages:

![Capture d'écran 2024-07-01 151309](https://github.com/Contributees/First-Ruby-Quest/assets/99920845/b9636249-8a8f-4c91-954b-9a5ae9658456)


![Capture d'écran 2024-07-01 151318](https://github.com/Contributees/First-Ruby-Quest/assets/99920845/1b5f4cc0-bec7-4d6c-b7d3-fd1fb4a90073)

![Capture d'écran 2024-07-01 151329](https://github.com/Contributees/First-Ruby-Quest/assets/99920845/77921b33-c8a2-472e-8f31-dc280b637e88)